osu.Game/Online/API/Requests/Responses/APIUser.cs

```diff
@@ -152,10 +152,10 @@ public class UserCover
         public int ScoresRecentCount;
 
         [JsonProperty(@"beatmap_playcounts_count")]
-        public int BeatmapPlaycountsCount;
+        public int BeatmapPlayCountsCount;
 
         [JsonProperty]
```

Comment:
```suggestion
        [JsonProperty(@"playstyle"]
```

Just to match the other ones in this file.